<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>967</width>
    <height>875</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>百年南开</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="7,1">
    <property name="rightMargin">
     <number>11</number>
    </property>
    <item>
     <widget class="QGraphicsView" name="graphicsView"/>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4" stretch="2,2,3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="styleSheet">
           <string notr="true">font: 10pt &quot;Staccato222 BT&quot;;
color: rgb(0, 0, 0);
</string>
          </property>
          <property name="text">
           <string>P1        </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="styleSheet">
           <string notr="true">font: 10pt &quot;Staccato222 BT&quot;;
color: rgb(0, 0, 0);
</string>
          </property>
          <property name="text">
           <string>HP:     </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="styleSheet">
           <string notr="true">font: 10pt &quot;Staccato222 BT&quot;;
color: rgb(0, 0, 0);
</string>
          </property>
          <property name="text">
           <string>Integral:  </string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="styleSheet">
           <string notr="true">font: 10pt &quot;Staccato222 BT&quot;;
color: rgb(0, 0, 0);
</string>
          </property>
          <property name="text">
           <string>P2     </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="styleSheet">
           <string notr="true">font: 10pt &quot;Staccato222 BT&quot;;
color: rgb(0, 0, 0);
</string>
          </property>
          <property name="text">
           <string>HP:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="styleSheet">
           <string notr="true">font: 10pt &quot;Staccato222 BT&quot;;
color: rgb(0, 0, 0);
</string>
          </property>
          <property name="text">
           <string>Integral:</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QPushButton" name="btn_start">
          <property name="styleSheet">
           <string notr="true">font: 10pt &quot;Staccato222 BT&quot;;
color: rgb(0, 0, 0);
</string>
          </property>
          <property name="text">
           <string>Start</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_pause">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 10pt &quot;Staccato222 BT&quot;;
color: rgb(0, 0, 0);
</string>
          </property>
          <property name="text">
           <string>Pause</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_stop">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 10pt &quot;Staccato222 BT&quot;;
color: rgb(0, 0, 0);
</string>
          </property>
          <property name="text">
           <string>Stop</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>967</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
